app/config/FrontendAppConfig.scala

```diff
@@ -31,9 +31,6 @@ class FrontendAppConfig @Inject() (val configuration: Configuration, val service
   lazy val citizenDetailsUrl: String       = configuration.get[Service]("microservice.services.citizen-details").baseUrl
   lazy val employeeWfhExpensesHost: String = servicesConfig.baseUrl("employee-wfh-expenses-frontend")
 
-  lazy val employeeTaxReliefExpensesClaimOnlineUrl: String =
-    configuration.get[String]("urls.employeeTaxReliefExpensesClaimOnlineUrl")
```

Comment:
If you're removing this config field, then please also remove the corresponding value in `application.conf`